api/v3/Order.php

```diff
@@ -91,9 +91,7 @@ function civicrm_api3_order_create($params) {
       if ($entityParams) {
         if (in_array($entity, ['participant', 'membership'])) {
           $entityParams['skipLineItem'] = TRUE;
-          if ($contributionStatus === 'Pending') {
-            $entityParams['status_id'] = ($entity === 'participant' ? 'Pending from incomplete transaction' : 'Pending');
-          }
+          $entityParams['status_id'] = ($entity === 'participant' ? 'Pending from incomplete transaction' : 'Pending');
```

Comment:
I think we can get away with this simplification as not-pending has been deprecated for 10 months